ingestion/src/metadata/utils/sql_queries.py

```diff
@@ -481,9 +481,9 @@
         JOIN pg_catalog.pg_stat_activity a ON d.datname = a.datname
```

Comment:
Can we please double-check that the JOIN logic here is correct? Looking at the results with a community user it seemed that we were getting quite old queries listed as new. cc @ulixius9

Reply:
Based on https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-ACTIVITY-VIEW, it seems that we are picking up `pg_stats_activity` information based on `datname`, which is the Name of the database. I believe we might need to instead JOIN here by `queryid` with any related table

Reply:
Hey @pmbrull, got a chance to review this and found that the logic of join implementation here is incorrect and regarding joining `pg_stats_activity` with `pg_stat_statements` via `queryid` is not possible because `pg_stats_activity` does not store all the query logs otherwise we would used that table itself, I've made changes in the code to remove this invalid implementation, in my brief research I found that fetching execution start time for  `pg_stat_statements` is not possible.. so this also removes the filtering on the query based on start time